src/it/scala/com/wavesplatform/it/ActivationTestSuite.scala

```diff
@@ -0,0 +1,17 @@
+package com.wavesplatform.it
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import org.scalatest.{BeforeAndAfterAll, FreeSpec, Matchers}
+
+import scala.concurrent.Await
+
+class ActivationTestSuite extends FreeSpec with Matchers with BeforeAndAfterAll with IntegrationNodesInitializationAndStopping {
+  override val docker = new Docker()
+  override val nodes = Docker.NodeConfigs.getConfigList("nodes").asScala.take(2).map(docker.startNode)
+
+  "api consuming example" in {
+    Await.result(nodes.head.waitForHeight(5), 5.minute)
```

Comment:
Use `Future.traverse` to await for all nodes to get to some state

Reply:
If you want to test particular node, what's the point to wait for others?